apps/cowswap-frontend/src/modules/hooksStore/dapps/PermitHookApp/hook.tsx

```diff
@@ -0,0 +1,16 @@
+import buildImg from './build.png'
+
+import { HookDappInternal, HookDappType } from '../../types/hooks'
+
+import { PermitHookApp } from './index'
+
+export const PRE_PERMIT: HookDappInternal = {
+  name: `Permit one token`,
+  descriptionShort: 'Infinite permit an address to spend one token on your behalf',
+  description: `This hook allows you to permit an address to spend your tokens on your behalf. This is useful for allowing a smart contract to spend your tokens without needing to approve each transaction.`,
```

Comment:
Not necessarily for this PR/feature, but we do have somewhat frequent requests to allow users to customize the approved amount.
Might be interesting use case for users to set their custom allowance with the permit hook.

Reply:
Humm interesting but I built this simple app just to show the new module and let you guys test it, so I think that we don't need to add it here